linux/networking-and-security/webscraping/limit-the-wget-download-rate.md

```diff
@@ -14,23 +8,6 @@ type: normal
 
 category: how to
 
-tags:
-
-  - linux
-
-  - wget
-
-  - rate
-
-  - limit
-
-  - terminal
-
-  - download
```

Comment:
no need to remove tags. although they are optional (and not used currently), we might use them in the future for search features

Reply:
Ok, ill add them back

Reply:
Done